src/card.tsx

```diff
@@ -72,16 +82,11 @@ const CardContent: React.FC<CardContentProps> = ({
                                 {renderHeadline()}
                                 <Stack space={4}>
                                     {pretitle && (
-                                        <Text1
-                                            truncate={pretitleLinesMax}
-                                            as="div"
-                                            regular
-                                            transform="uppercase"
-                                        >
+                                        <Text1 truncate={pretitleLinesMax} as="div" regular>
                                             {pretitle}
                                         </Text1>
                                     )}
-                                    <Text4 truncate={titleLinesMax} as="h1" regular>
+                                    <Text4 truncate={titleLinesMax} as="h3" regular>
```

Comment:
why h3?

Reply:
it seems no browser support h1 inside article/section to create a new hierarchy. So, maybe h3 is a good choice for cards

Reply:
Would't be too risky to make that assumption? A card can be the only element in a main section without a title (h2), or inside a subsection (h4), since the semantic hierarchy of an element only can be defined based on the layout that element is placed, maybe we shoud expose the title hierarchy? Or...avoid using a heading for the moment?

Reply:
yes, it won't be the perfect h in all cases, but I think it would be worse not having an h at all